eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1476,6 +1476,13 @@
     <Bug pattern="RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT"/>
   </Match>
 
+  <!--  ChannelHandlerContext::fireUserEventTriggered is fluent method and return ref can be ignored -->
+  <Match>
+    <Class name="com.azure.cosmos.implementation.directconnectivity.rntbd.RntbdClientChannelPool"/>
```

Comment:
DM_exit exception is missing for

"com.azure.cosmos.test.implementation.ImplementationBridgeHelpers$FaultInjectionRuleHelper",32,"com.azure.cosmos.test.implementation.ImplementationBridgeHelpers$FaultInjectionRuleHelper.getFaultInjectionRuleAccessor() invokes System.exit(...), which shuts down the entire virtual machine","DM_EXIT"

Reply:
I added the change in the target folder :(

Reply:
updated